hathorlib/conf/settings.py

```diff
@@ -44,3 +44,6 @@ class HathorSettings(NamedTuple):
 
     # Amount in which tx min weight reaches the middle point between the minimum and maximum weight
     MIN_TX_WEIGHT_K: int = 100
+
+    # Maximum size of the tx output's script allowed for a tx to be standard
+    MAX_OUTPUT_SCRIPT_SIZE: int = 256
```

Comment:
This name has a conflict with another constant in hathor-core. I think we should rename it to `MAX_STD_OUTPUT_SCRIPT_SIZE`. And I don't remember where we enforce this rule in hathor-core and tx-mining-service.

Reply:
@jansegre I don't understand the part of preserving the compatibility. This is a new constant and will be used not only for pushtx but also for mining tx. Maybe change to `MAX_STD_OUTPUT_SCRIPT_SIZE` as @msbrogli suggested?

Reply:
As it is now, `hathorlib/conf/get_settings.py` will try to load the config file from the env var `TXMINING_CONFIG_FILE`. If we want to make this configurable in `hathor-core`, we will be in a weird place where we have to set _that_ variable to customize it. If we eventually want to converge both settings in a backwards compatible way, they'll have to use the same names for the same settings.

Reply:
Ahhh makes sense. I think I agree with you to use `PUSHTX_MAX_OUTPUT_SCRIPT_SIZE` to keep compatibility with hathor-core settings. And we can have future refactor to break this compatibility